demos/swift/gen_swift.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+
+global.FOAM_FLAGS = {
+  'node': true,
+  'swift': true,
+  'debug': true,
+};
+
+var execSync = require('child_process').execSync
+
+var dir = __dirname;
+var root = dir + '/../..';
+var genDir = dir + '/gen';
```

Comment:
I think we should pass the output directory in as a command line parameter like we do for genjava.js

Also would be useful to take a classes.js file like we do for genjava.js

Reply:
Sounds good. Created a tool similar to genjava.js that this script now makes use of.